code/datums/components/radioactive.dm

```diff
@@ -63,16 +63,16 @@
 	var/list/out = list()
 	if(get_dist(master, user) <= 1)
 		out += "The air around [master] feels warm"
-	switch(strength)
-		if(RAD_AMOUNT_LOW to RAD_AMOUNT_MEDIUM)
-			out += "[out ? " and it " : "[master] "]feels weird to look at."
-		if(RAD_AMOUNT_MEDIUM to RAD_AMOUNT_HIGH)
-			out += "[out ? " and it " : "[master] "]seems to be glowing a bit."
-		if(RAD_AMOUNT_HIGH to INFINITY) //At this level the object can contaminate other objects
-			out += "[out ? " and it " : "[master] "]hurts to look at."
-		else
-			out += "."
-	to_chat(user, out.Join())
+		switch(strength)
+			if(RAD_AMOUNT_LOW to RAD_AMOUNT_MEDIUM)
+				out += "[out ? " and it " : "[master] "]feels weird to look at."
```

Comment:
You can fix this properly by doing `length(out)` instead in the ternary

Reply:
I don't see why it's necessary; I overlooked (and fixed) a minor issue that would cause a missing fullstop but otherwise this seems fully functional and reasonably compact.

Reply:
Because that way you keep the original behavior and only fix the bug instead of also introducing a change

Reply:
Oh, it's _supposed_ to display those messages at a distance? Sorry, i'll fix it properly then.

Reply:
Alright, done.

Reply:
Update changelog to reflect this please